app/views/gobierto_dashboards/dashboards/contracts.html.erb

```diff
@@ -12,6 +12,10 @@
     </div>
   </div>
 
+  <div id="loading">
```

Comment:
Quite general. Indeed, it's very likely something like this is already defined somewhere else. Two things here:

1. What if you load the Vue app with no data. Loading becomes a Vue component (shared). Once the data is fetched, you hide the Loading. In that way, you won't mix "reactive stuff" with static views.

2. If you need to access to the DOM, a good practice is about adding a css class like `js-loading` instead of using a very generic name

Reply:
Thanks, I changed it so it uses a css class.

I saw the Loading component was already done in other modules, but those were getting the data within the vue app. Since the data here is fetched from outside vue, I thought this would be a more straightforward solution.

Reply:
That's why I say to load the full app with no data, an empty array. Then, when it's populated, you pass the data as a prop, and handle the states as usual